line-sdk/src/main/java/com/linecorp/linesdk/api/internal/LineApiClientImpl.java

```diff
@@ -219,56 +216,68 @@ public LineApiClientImpl(
                                                         boolean isForOttShareMessage) {
         return callWithAccessToken(
-                accessToken -> talkApiClient.getGroups(
-                        accessToken,
-                        nextPageRequestToken,
-                        isForOttShareMessage
-                )
-        );
+            accessToken -> talkApiClient.getGroups(accessToken, nextPageRequestToken, isForOttShareMessage));
     }
 
     @Override
     @TokenAutoRefresh
     @NonNull
     public LineApiResponse<GetFriendsResponse> getGroupApprovers(@NonNull String groupId,
                                                                  @Nullable String nextPageRequestToken) {
-        return callWithAccessToken(
-                accessToken -> talkApiClient.getGroupApprovers(accessToken, groupId, nextPageRequestToken)
-        );
+        return callWithAccessToken(accessToken -> talkApiClient.getGroupApprovers(accessToken, groupId, nextPageRequestToken));
     }
 
     @Override
     @TokenAutoRefresh
     @NonNull
     public LineApiResponse<String> sendMessage(@NonNull String targetUserId,
                                                @NonNull List<MessageData> messages) {
-        return callWithAccessToken(
-                accessToken -> talkApiClient.sendMessage(accessToken, targetUserId, messages)
-        );
+        return callWithAccessToken(accessToken -> talkApiClient.sendMessage(accessToken, targetUserId, messages));
     }
 
     @Override
     @TokenAutoRefresh
     @NonNull
     public LineApiResponse<List<SendMessageResponse>> sendMessageToMultipleUsers(
-            @NonNull List<String> targetUserIds,
-            @NonNull List<MessageData> messages
-    ) {
+        @NonNull List<String> targetUserIds,
+        @NonNull List<MessageData> messages
+                                                                                ) {
         return sendMessageToMultipleUsers(targetUserIds, messages, false);
     }
 
     @Override
     @TokenAutoRefresh
     @NonNull
     public LineApiResponse<List<SendMessageResponse>> sendMessageToMultipleUsers(
-            @NonNull List<String> targetUserIds,
-            @NonNull List<MessageData> messages,
-            boolean isOttUsed
-    ) {
+        @NonNull List<String> targetUserIds,
+        @NonNull List<MessageData> messages,
+        boolean isOttUsed
+                                                                                ) {
         return callWithAccessToken(
-                accessToken -> talkApiClient.sendMessageToMultipleUsers(
-                        accessToken, targetUserIds, messages, isOttUsed)
-        );
+            accessToken -> talkApiClient.sendMessageToMultipleUsers(accessToken, targetUserIds, messages, isOttUsed)
+                                  );
+    }
+
+    @NonNull
+    public LineApiResponse<Boolean> updateAgreementStatus(@NonNull Boolean agreed) {
```

Comment:
From the function name, we don't know what kind of agreement status we want to update. I think it will be more accurate if we rename it to `updateOpenChatAgreementStatus()`.